lib/ash_json_api/json_schema/open_api.ex

```diff
@@ -1759,7 +1764,20 @@ if Code.ensure_loaded?(OpenApiSpex) do
             if route.type == :route do
               []
             else
-              [include_parameter(resource), fields_parameter(resource)]
+              # included_page_parameter is only for routes that return collections (index, get, related)
+              # include and fields can be used on any route to control response format
+              if route.type in [:index, :get, :related, :get_related] do
```

Comment:
One other thing, this needs to only show if the resource has at least one paginatable relationship.